src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -196,6 +196,11 @@
       <arg name="text" type="s" direction="out"/>
     </signal>
 
+    <!-- Straight from your door to our console, unadulterated! -->
+    <signal name="RawMessage">
```

Comment:
I think the problem is that if we stop emitting `Message` and start emitting `RawMessage` it will mean clients like Cockpit stop getting them.

Maybe take a step back and look at what messages we are emitting now.  If all we want is:

`<RawMessage>Frobnicating the baz...</><RawMessage>done!\n</>` that might be simpler with something like `<BeginTask>Frobnicating the baz...</><EndTask/>` ?